examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p><code>DEVICES = [5, 6, 7]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;# GPUs to use</code></p><p> </p><p><code>TEST_FREQUENCY = 150              # how often model evaluation on test data is done during train</code></p><p> </p><p>~~~</p><p> </p><p>Also, change <code>RESEARCH_NAME</code> variable value to be more descriptive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>